src/options/options.html

```diff
@@ -59,110 +59,153 @@
-								<label data-i18n="__MSG_optionQrCodeTypeSvg__" for="qrCodeTypeSvg">SVG</label>
-								<span data-i18n="__MSG_optionQrCodeTypeSvgHelper__" class="helper-text">Uses a modern generation method with vector graphics that enable smoother resizing.</span>
-							</li>
-
-							<li class="condensed-list">
-								<input id="qrCodeTypeCanvas" type="radio" name="qrCodeType" value="canvas" class="">
-								<label data-i18n="__MSG_optionQrCodeTypeCanvas__" for="qrCodeTypeCanvas">Canvas image</label>
-								<span data-i18n="__MSG_optionQrCodeTypeCanvasHelper__" class="helper-text">A legacy generation method generating the QR code as an image ("canvas").</span>
-							</li>
-						</ul>
-					</fieldset>
-				</li>
-				<li>
-					<fieldset id="sizeType" data-optiongroup="qrCodeSize" data-type="radiogroup" class="setting">
-						<legend data-i18n="__MSG_optionQrCodeSize__">QR code size</legend>
-
-						<ul>
-							<li class="condensed-list">
-								<input id="qrCodeSizeFixed" type="radio" name="sizeType" value="fixed" class="save-on-input">
-								<label data-i18n="__MSG_optionQrCodeSizeFixed__" for="qrCodeSizeFixed">Fixed size: </label>
-
-								<input data-i18n="__MSG_optionPixelAbbreviation__" class="setting save-on-change" data-optiongroup="qrCodeSize" type="number" id="qrCodeSizeFixedValue" min="160" step="5" size="5" name="size">
-								<label for="qrCodeSizeFixedValue">px</label>
-							</li>
-
-							<li class="condensed-list">
-								<input id="qrCodeSizeAuto" type="radio" name="sizeType" value="auto" class="save-on-input">
-								<label data-i18n="__MSG_optionQrCodeSizeAuto__" for="qrCodeSizeAuto">Automatically adjust size</label>
-								<span data-i18n="__MSG_optionQrCodeSizeAutoHelper__" class="helper-text">Mostly useful on mobile devices, where the QR code opens in a new tab.</span>
-							</li>
-
-							<li class="condensed-list">
-								<input id="qrCodeSizeRemember" type="radio" name="sizeType" value="remember" class="save-on-input">
-								<label data-i18n="__MSG_optionQrCodeSizeRemember__" for="qrCodeSizeRemember">Remember last size</label>
-							</li>
-						</ul>
-					</fieldset>
-				</li>
-				<li>
-					<label data-i18n="__MSG_optionQrQuietZone__" for="qrQuietZone">QR quiet zone: </label>
-					<input class="setting save-on-input" type="range" value="1" min="0" max="5" step="1" id="qrQuietZone" name="qrQuietZone">
-					<span>
-						<span id="qrQuietZoneStatus">(1 square)</span>
-					</span>
-					<span class="helper-text">
+			<section class="mobile-incompatible">
+				<h1 class="first" data-i18n="__MSG_titleGeneral__">Add On</h1>
+				<ul>
+					<li>
+						<input class="setting save-on-change" type="checkbox" id="popupIconColored" name="popupIconColored">
+						<label data-i18n="__MSG_optionPopupIconColoredDescr__" for="popupIconColored" >Shows a colored icon instead of the black/white icon in the toolbar.</label>
```

Comment:
BTW, as for this option, I think it sounds strange now (the label is kinda elaborate). Remember, before it was a label + a helper text:
![grafik](https://user-images.githubusercontent.com/11966684/54934926-955d2200-4f1f-11e9-82a9-7ecaed158716.png)

Maybe we can restore it like this?